### PR TITLE
feature/tokenListItem DAO-538 

### PR DESCRIPTION
**Pull request by sepehr2github**
- Token List item added

### CODE REVIEW

**Comment by RakeshUP** [comment]
There's a component already tokenCard that looks similar. Do you think the tokenCard could be modified to make it compatible with the requirements of this PR DAO-538? @sepehr2github

**Comment by sepehr2github** [comment]
> 

Thank you Rakash For this suggestion. but TokentemList is a button and has a different structure than tokenCard which is a div element but We can change it later if needed

File: packages/ui-components/src/components/tokenListItem/tokenListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Avatar} from '../avatar';
+
+export type TokenListItemProps = {
+  /**
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/tokenListItem/tokenListItem.tsx:6]: Don't forget the `bgWhite` prop!

File: packages/ui-components/src/components/tokenListItem/tokenListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const Container = styled.button.attrs(({disabled}: StyledContentProps)=> ({
+  className:`w-full flex justify-between items-center py-1.5
+  px-2 hover:text-ui-800 hover:bg-ui-100 active:text-ui-800 
+  active:bg-ui-200 disabled:text-ui-300 rounded-xl
+  ${disabled ? 'bg-ui-100' : 'bg-white'}`,
+}))``;
+
+const AmountWrapper = styled.h3.attrs({
+  className:'font-semibold text-ui-600 text-base',
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/tokenListItem/tokenListItem.tsx]: I think the font weight has changed to 500 so normal.

**Review by Fabricevladimir — APPROVED** [review]
Nice, nice!

**Comment by RakeshUP** [comment]
> > 
> 
> Thank you Rakash For this suggestion. but TokentemList is a button and has a different structure than tokenCard which is a div element but We can change it later if needed

Sorry I completely missed that fact that this component is a button!

**Review by RakeshUP — APPROVED** [review]

**Pull request merged**